src/main/java/graphql/nadel/NadelExecutionInput.java

```diff
@@ -103,9 +111,13 @@ public Builder artificialFieldsUUID(String artificialFieldsUUID) {
             return this;
         }
 
+        public Builder nadelExecutionHints(NadelExecutionHints nadelExecutionHints) {
+            this.nadelExecutionHints = nadelExecutionHints;
```

Comment:
`this.nadelExecutionHints = GraphqAssert.assertNotNull(nadelExecutionHints)`

Reply:
Elsewhere in this file (NadelExecutionInput) we use `java.util.Objects.requireNonNull`. Is there a difference between the GraphQL assert and the Java util non null check?